### ISSUE TITLE
Improve accessibility for banner

### ISSUE THREAD

**Issue by maliknur** [BranchMetrics/web-branch-deep-linking#457]
Currently banner has some accessibility issues for screen readers, especially `x` close button which has been implemented with non standard way using `div` tag and Screen reader like 'Voice over' on iPhone reads it as `times` or `multiplication sign` instead of `close`. So users who use assistive technologies have problems to understand that element on banner. 

For this purpose, [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) can be used which is  designed to help assistive technology (e.g. screen readers) attach a label to an otherwise anonymous HTML element.

Also icon which is used in banner doesn't have `alt` attribute and screen reader reads file name, which can be improved with 'alt="Application icon"'.

Repro steps:
1. Turn on VoiceOver or Talkback depending on device (ios/android)
2. Launch any banner enabled demo
3. Navigate to banner and try to close banner

![img_1445 copy](https://cloud.githubusercontent.com/assets/12554919/26600468/08e0b8ae-4531-11e7-8d4c-fd41d136e050.PNG)

I've made a small pull request for review.

**Comment by maliknur** [comment]
Closing issue since PR has been merged.
